bucket/haskell.json

```
@@ -29,14 +29,14 @@
     ],
     "env_add_path": "lib\\bin",
     "checkver": {
-        "url": "https://api.github.com/repositories/46509594/tags",
-        "jsonpath": "$..name",
-        "regex": "ghc-([\\d\\.]+)-release"
+        "url": "https://downloads.haskell.org/~ghc/",
+        "regex": "(\\d\\.\\d\\.\\d)/",
```

What about the x.x.x.YYYYMMDD releases? Or if a hotfix to 9.8.2 is released like 9.8.2.1? This regex will miss them out. Maybe it's better to use the release posted as the current stable in https://www.haskell.org/ghc/download.html

---

```suggestion
        "url": "https://downloads.haskell.org/~ghc/",
        "regex": ">([\\d.]+)/",
```
How about this to catch any number of point versions? This is the standard regex we use without catching versions with words in it like alpha, beta, etc.

---

Wouldn't this allow `..` to be parsed as a valid version?
Maybe something like `>(\\d(\\.\\d)*)/`?

---

I'll push my proposed regex so you can merge it quickly in case you agree with it, if not, I'll be happy to swap it out again. 😊